lib/verbalize.rb

```diff
@@ -79,13 +88,16 @@ def __verbalized_send(method_name, *args)
       end
 
       Failure.new(error)
+    rescue VerbalizeError => e
+      Failure.new(e.failure)
```

Comment:
If we wrap the failure value in `fail!` instead of here, then the return value is just the result of the `catch` or `e.failure` here.